In [1]:
import pandas as pd

In [2]:
nstemi = pd.read_csv('raw_data/nstemi.csv')

In [3]:
stemi = pd.read_csv('raw_data/stemi.csv')

# Reviewer 1 - Q1

How many patients underwent contrast enhanced TTE or cardiac MRI due to poor image quality? 

Key:
-	0: TTE 
-	1: TEE 
-	2: CT 
-	3: MRI
-	4: Cardiac Ventriculography

No data available on contrast enhancement

In [5]:
nstemi['InitialScan'].value_counts()

0    76
2     1
1     1
Name: InitialScan, dtype: int64

In [6]:
stemi['InitialScan'].value_counts()

0    208
2      3
Name: InitialScan, dtype: int64

In [8]:
nstemi['FinalScan'].value_counts()

999    46
0      30
3       2
Name: FinalScan, dtype: int64

In [9]:
stemi['FinalScan'].value_counts()

0      129
999     71
3       10
1        1
Name: FinalScan, dtype: int64

NSTEMI:
Initial - TTE = 76, TEE = 1, CT = 1
Final - TTE = 30, MRI = 2

STEMI
Initial - TTE = 208, CT = 1
Final - TTE = 129, TEE = 1, MRI = 10 

Total:
Initial - TTE = 284, TEE = 1, CT = 1
Final - TTE = 159, TEE = 1, MRI = 12